make_html.py

```diff
@@ -17,6 +21,21 @@
 
 from htmlify_comment import htmlify_comment
 
+if not os.path.exists('build/repos/mathlib'):
+    print("Make sure you are in the root of the mathlib-port-status directory")
+    print("and have checked out mathlib under build/repos/mathlib.")
+    sys.exit(1)
+
+GITHUB_TOKEN_FILE = 'build/repos/github-token'
```

Comment:
I think it makes sense to load this from an env var, which is then easy to set in the workflow.yaml.

Also, if the var is missing I think you can just use `None`, and you'll just be rate-limited a little more

Reply:
reading from enviroment variable `GITHUB_TOKEN_FILE` now, otherwise use None....